docs/windows.md

```diff
@@ -0,0 +1,158 @@
+##### Discovering available devices:
```

Comment:
With Windows, did you use Visual C++ or mingw ? I believe there were a few lines in the previous Windows specific ReadMe that included some Visual C++ and mingw related details. Can we bring those in as well if we still continue to use it with the new repo? 


Rest looks good to me! Great work!